api.ts

```diff
@@ -399,11 +399,8 @@ export type BooleanPackFormula<ParamDefsT extends ParamDefs> = BaseFormula<Param
   schema?: BooleanSchema;
 };
 
-export type StringPackFormula<
-  ParamDefsT extends ParamDefs,
-  ResultT extends StringHintTypes = StringHintTypes,
-> = BaseFormula<ParamDefsT, SchemaType<StringSchema<ResultT>>> & {
-  schema?: StringSchema<ResultT>;
+export type StringPackFormula<ParamDefsT extends ParamDefs> = BaseFormula<ParamDefsT, SchemaType<StringSchema>> & {
```

Comment:
@huayang-codaio Should check that this doesn't bust anything with the strong typing for return values he just added.

Reply:
seems like `SchemaType<StringSchema>` is identical to string here anyway.